### PR TITLE
Add info that was omitted

### PR DESCRIPTION
**Pull request by qwrtln**
Added missing info about removing cards and secondary heroes using the same units. Also, adjusted the layout of combat a little, and moved some art around.

Technical:
 - bumped version to 1.3, it should be released soon
 - collapsed po files in PRs by default

Diff:
en-15
en-18
en-33
en-34

Polish:
pl-15
pl-18
pl-33
pl-34

### CODE REVIEW

File: sections/deckbuilding.tex [hunk 1]
```diff
@@ -30,6 +30,7 @@ \subsection*{General Card Rules}
     \item \textbf{Permanent} \includesvg[height=10px]{\svgs/permanent.svg} Cards\index{Permanent Cards} stay in play until discarded or replaced.
       \textbf{You may only have one permanent Card at a time}; playing another discards the first.
   \end{itemize}
+  \item Whenever you are instructed to \textbf{remove a Card}, remove it completely from the game.
```
Comment (DarthGandalf) [inline sections/deckbuilding.tex:33]: Perhaps mention the tournament rules here which modify this?

**Review by DarthGandalf — APPROVED** [review]

File: sections/deckbuilding.tex (hunk 1 above)
Comment (qwrtln) [reply sections/deckbuilding.tex:33]: Not sure which rules you have in mind. Do we mention them anywhere?

Comment (DarthGandalf) [reply sections/deckbuilding.tex:33]: Выделение_610

It's in the tournament book

**Review by Heegu-sama — APPROVED** [review]

File: sections/deckbuilding.tex (hunk 1 above)
Comment (qwrtln) [reply sections/deckbuilding.tex:33]: Not sure if it belongs here. Since it's a separate book with separate rules, maybe they should stay there?

**Pull request merged**